### ISSUE TITLE
Invalid code signing entitlements

### ISSUE THREAD

**Issue by olokobayusuf** [fxnai/fxnc#2]
`Function.dylib` has an invalid `com.apple.application-identifier` (using `codesign -d --entitlements :-`):
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "https://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <key>com.apple.application-identifier</key>
    <string>NPA3XMW8F9.</string>
  </dict>
</plist>
```
As such macOS apps submitted to App Store Review are being rejected.

**Comment by marvpaul** [comment]
Thanks for investigating. Any ETA for this? Would be great to upload a new version of my app to the AppStore as soon as possible as it includes critical bugfixes.

**Comment by olokobayusuf** [comment]
Hey @marvpaul ETA for this is Monday. We're working a large update that includes this fix. Sit tight.

**Comment by marvpaul** [comment]
@marvpaul ah great, looking forward to it!

**Comment by olokobayusuf** [comment]
@marvpaul almost done with the update, hoping to get it out today.

**Comment by olokobayusuf** [comment]
Fixed in [0.0.30](https://github.com/fxnai/fxnc/releases/tag/0.0.30). The updated library will be propagated downstream (Python, Unity SDKs) over the coming days. Thanks for bringing this to our attention!

**Comment by olokobayusuf** [comment]
Reopening this issue. From Apple's reference, it looks like we have to remove the code signing entitlements entirely (versus update the format to be compliant with Apple's requirement, which is what we did initially). Working on a fix right now.

**Comment by olokobayusuf** [comment]
[Fixed in 0.0.31](https://github.com/fxnai/fxnc/releases/tag/0.0.31). Will propagate downstream to Unity package in coming days.
